common/tests/crypto/testCrypto.cpp

```diff
@@ -472,11 +494,13 @@ int pcrypto::testCrypto()
     }
     catch (const Error::ValueError& e)
     {
-        SAFE_LOG(PDO_LOG_DEBUG, "testCrypto: RSA invalid public key deserialize detected!\n%s\n", e.what());
+        SAFE_LOG(PDO_LOG_DEBUG, "testCrypto: RSA invalid public key deserialize detected!\n%s\n",
+            e.what());
     }
```

Comment:
i know this was from clangformat, but there is a lot of inconsistency in how the lines are broken... some after the log level some after the string. consistency would be very nice.

Reply:
I agree it's ugly, and I don't have an answer on why that happens.
Anyway, I made 2 modifications to clang:
1. if parameters do not fit in the line, they will be put on separate lines
2. when on separate lines, they will be aligned with the parenthesis

I have only ran clang on the files in this PR -- we will have to run it over the entire repo.